pages/components/grid/grid.css

```diff
@@ -0,0 +1,55 @@
+.grid {
+    display: grid;
+    gap:0.75rem;
+    width: 45rem;
+    height: 7.5rem;
+}
+
+.grid_two_items {
+    grid-template-areas: 
+    "Item_one Item_two"
+    "Item_three Item_four";
+}
+
+.grid_three_items {
+    grid-template-areas: 
+    "Item_one Item_two Item_three"
+    "Item_four Item_five Item_six";
+}
+
```

Comment:
same here.

Reply:
modified class names to give better meanings. separated width, height from 'grid' class.